src/fsharp/vs/ServiceUntypedParse.fs

```diff
@@ -340,8 +340,13 @@ type FSharpParseFileResults(errors : FSharpErrorInfo[], input : Ast.ParsedInput
  
         ErrorScope.Protect 
             Range.range0 
-            (fun () -> findBreakPoints() |> List.tryLast)
-            (fun _msg -> None)   
+            (fun () -> 
+                let locations = findBreakPoints()
+                
+                match locations |> List.filter (fun m -> rangeContainsPos m pos) with
+                | [] -> Seq.tryHead locations
```

Comment:
Is it not possible for a line to have no breakpoints allowable?  Like inside a multi-line comment for example.

If the range doesn't contain pos, shouldn't the result be None, rather than the first entry in the locations list even though that entry is not in the range?